source/getting-started/routes-and-templates.md

```diff
@@ -0,0 +1,166 @@
+
+![super rentals homepage screenshot](../../images/routes-and-templates/ember-super-rentals-index.png)
+
+We arrive at the home page which shows a list of rentals.  From here, we will be able to navigate to an about page and a contact page.  
+
+We already have the new app called super-rentals created. Before we start building the three pages for our app, we are going to clear out the contents of the `app/templates/application.hbs` file and only leave the {{outlet}} code in place.  We'll talk more about the role of the `application.hbs` file after our site has a few pages.
+
+Now, let's start with building our "about" page. Remember, when the URL path `/about` is loaded, the router will map the URL to the route handler of the same name, _about.js_.  The route handler then loads a template.  
+
+## An About Route
+
+If we run `ember help generate`, we can see a variety of tools come with Ember for automatically generating files for various Ember resources.  We'll use the route generator to build our `about` route.
+
+```shell
+ember generate route about
+```
+
+or for short,
+
+```shell
+ember g route about
+```
+
+We can see what actions are taken by the generator:
+
+```shell
+version: 1.13.8
+installing route
+  create app/routes/about.js
+  create app/templates/about.hbs
+updating router
+  add route about
+installing route-test
+  create tests/unit/routes/about-test.js
+```
+
+Three new files are created: one for the route handler, one for the template the route handler will render, and a test file.  The fourth file that is touched is the router.  
+
+When we open the router, we can see that the generator has mapped a new _about_ route for us. This route will load the `about` route handler.
+
+```app/router.js
+import Ember from 'ember';
+import config from './config/environment';
+
+var Router = Ember.Router.extend({
+  location: config.locationType
+});
+
+Router.map(function() {
+  this.route('about');
+});
+
+export default Router;
+```
+
+By default, the `about` route handler loads the `about.hbs` template. So we don't actually have to change anything in the new `app/routes/about.js` file for the `about.hbs` template to render as we want.
+
+With all of the routing in place from the generator, we can get right to work on coding our template.  For our `about` page, we'll add some HTML that just says a little about the site:
+
+```app/templates/about.hbs
+<h2>About Super Rentals</h2>
+
+<p>The Super Rentals website is a delightful project created to explore Ember.<br>
+  By building a property rental site, we can simultaneously imagine traveling<br>
+  AND building Ember applications simultaneously.</p>
+```
+
+## A Contact Route
+
+Let's create another static page with details for contacting the company.  Once again, we'll start by generating a route, a route handler, and a template.
```

Comment:
I'd refrain from calling them static pages, as that would imply they're page.html files in `/public`.